imports/plugins/included/ui-search/client/templates/searchModal/sortFilter.html

```diff
@@ -0,0 +1,55 @@
+<template name="sortFilter">
+	<div id="sortFilter">
+		<div class="container sort-filter">
+			<div class="filter-div">
+				<label class="transform">Filter by Price</label>
+				<div class="rui select">
+					<select id="price-filter">
+						<option value="all" selected>All</option>
+						<option value="0-50">below $50</option>
+						<option value="50-100">$50 - $100</option>
+						<option value="100-500">$100 - $500</option>
+						<option value="500-1000">$500 - $1000</option>
+						<option value="1000-above">$1000 - above</option>
+					</select>
+				</div>
+			</div>
+
+			<div class="filter-div">
+				<label class="transform">Filter by Brand</label>
+				<div class="rui select">
+					<select id="brand-filter">
+						<option value="all" selected>All</option>
+						{{#each vendor in getVendors}}
+						<option value="{{vendor}}">{{vendor}}</option>
+						{{/each}}
+					</select>
+				</div>
+			</div>
+      <!-- <div class="sort-div" id="sortRatings">
+				<label>Filter by Ratings: </label>
+				<div class="rui select">
+					<select id="sort-value">
+						<option value="*">Any</option>
+						<option value="5">5 stars</option>
+						<option value="4">4 stars</option>
+            <option value="3">3 stars</option>
+						<option value="2">2 stars</option>
+            <option value="1">1 stars</option>
+					</select>
+				</div>
+			</div> -->
```

Comment:
Unused code. Maybe remove instead of commenting out, to uphold coding best practices.